### PR TITLE
test(inline-inputs): migrate tests from cypress to playwright 

### PR DESCRIPTION
**Pull request by ZhuoyuJin**
### Proposed behaviour

- Refactor cypress tests to playwright

### Current behaviour

- Tests are currently using cypress

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Playwright automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Related docs have been updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Additional context

N/A

### Testing instructions

- [x] Run Inline-inputs tests in Playwright Test Runner to check if the inline-inputs.pw.tsx file passed
- [x] Run all other tests in Playwright Test Runner to check none of the other *.pw.tsx files have regressed

### CODE REVIEW

**Comment by divyajindel** [comment]
Hi @ZhuoyuJin , Looks like you forgot to create "component.test-pw.tsx" file for mock components. so could you please create one top use in "src/components/inline-inputs/inline-inputs.pw.tsx"

**Comment by ZhuoyuJin** [comment]
> Hi @ZhuoyuJin , Looks like you forgot to create "component.test-pw.tsx" file for mock components. so could you please create one to use in "src/components/inline-inputs/inline-inputs.pw.tsx"

Hi Divya, thanks for your comment, I don't think we need to create the file "component.test-pw.tsx" as there's no mock components that needs move to.

File: src/components/inline-inputs/inline-inputs.pw.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+import { test, expect } from "@playwright/experimental-ct-react17";
+import Textbox from "../textbox";
+import Decimal from "../decimal";
+import InlineInputs, { InlineInputsProps } from "./inline-inputs.component";
+import {
+  Default,
+  WithAdaptiveLabelBreakpoint,
+  Required,
+} from "../../../src/components/inline-inputs/inline-inputs.stories";
+import { Default as InlineInputComponent } from "./inline-inputs-test.stories";
```
Comment (divyajindel) [inline src/components/inline-inputs/inline-inputs.pw.tsx:11]: we are not supposed to use this file to import mock components.

File: src/components/inline-inputs/inline-inputs.pw.tsx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    page,
+  }) => {
+    await mount(<Required />);
+
+    await checkAccessibility(page);
+  });
+});
+
+test.describe("rounded corners", () => {
+  ([
```
Comment (edleeks87) [inline src/components/inline-inputs/inline-inputs.pw.tsx:149]: comment (non-blocking): If you go with my suggestion above this and the test below could be replaced with `gutters.forEach`

**Review by edleeks87 — COMMENTED** [review]
I'm inclined to agree with @divyajindel , it doesn't look like there's any need to have the test stories anymore and whilst this works by importing from the story file I think for consistencies sake we should move it to a `components.test-pw` file but this is non-blocking for me so happy for it to be left if you disagree


**Update**

It looks like we've got a few components that have done the same thing so let's leave it for now and have a wider discussion as a team at a later date 👍

**Review by edleeks87 — APPROVED** [review]

**Review by divyajindel — APPROVED** [review]

**Pull request merged**